app/scripts/modules/core/pipeline/config/actions/pipelineConfigActions.html

```diff
@@ -6,7 +6,6 @@
     <li><a href ng-click="pipelineConfigurerCtrl.renamePipeline()">Rename</a></li>
     <li><a href ng-click="pipelineConfigurerCtrl.deletePipeline()">Delete</a></li>
     <li><a href ng-click="pipelineConfigurerCtrl.editPipelineJson()">Edit as JSON</a></li>
-    <li ng-if="pipeline.parallel"><a href ng-click="pipelineConfigurerCtrl.disableParallel()">Disable parallel stage execution</a></li>
```

Comment:
Should remove the method from the controller, too, since it won't be accessible.